speech/__init__.py

```diff
@@ -166,8 +172,8 @@ def handle_endtag(self, tag: str):
                 dattrs = self._attr_stack.pop()  # Current tag attributes
                 t: Optional[str] = dattrs.pop("type")
                 assert t, f"Missing type attribute in <greynir> tag around string: {s}"
-                # Fetch handler
-                hf = self._handlers.get(t)
+                # Fetch handler function
+                hf = getattr(self._handler, t)
```

Comment:
Is hf typed somewhere?

Reply:
It's a bit tricky to type hint this, I added a simple `(...) -> str` type for now.
I couldn't find a way to correctly specify "_classmethod which receives strings as keyword arguments_", but I'll fix this in the next pull request (which I'm currently working on).